### DISCUSSION TITLE
how to set Service Group for Service Mesh in UI

### DISCUSSION THREAD

**Discussion by devaDong** [apache/skywalking#9768] [Q&A]
Skywalking v 9.0.0
I want to observe Service Mesh in Skywalking, But I don't know where to set Service Group for Service Mesh?
After several rounds of searching for Skywalking's official website, I still can't find it.

**Comment by wu-sheng** [comment]
Isn't the doc right in place? https://skywalking.apache.org/docs/main/next/en/setup/backend/service-auto-grouping/

First doc of setup. Don't know what you mean `can't find it`. Have you ever read the whole doc?

**Reply by devaDong** [reply]
I know this place . https://skywalking.apache.org/docs/main/next/en/setup/backend/service-auto-grouping/
I have already set the group for normal Service. But it doesn't seem to work for the Service Mesh.
In this place(https://skywalking.apache.org/docs/main/next/en/setup/envoy/metrics_service_setting/ ) ,there is no suitable parameter to set the Group Name and Service Name for Data Plane.

**Reply by wu-sheng** [reply]
If it is about service mesh in practice, check the showcase setup. It is there.
